src/components/profile/ProfileSection.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import { ScrollView } from "react-native";
+import { getData } from "@govtechsg/open-attestation";
+import { ProfileImage } from "./ProfileImage";
+import { ProfileBasicDetails } from "./ProfileBasicDetails";
+import { ProfilePassDetails } from "./ProfilePassDetails";
+import { VerifyProfile } from "./VerifyProfile";
+import { VerifyModal } from "../Modals/VerifyModal";
+
+const ProfileSection = ({ workpass, navigation, isPreview }) => {
+  const cleanDocument = getData(workpass);

```

Comment:
```suggestion
  const { pass, recipient, employer } = getData(workpass);
```

You wont be using cleanDocument, so might as well...

Reply:
done